packages/core/src/browser/authentication-service.ts

```diff
@@ -27,12 +27,8 @@ import { Disposable } from '../common/disposable';
 import { ACCOUNTS_MENU, ACCOUNTS_SUBMENU, MenuModelRegistry } from '../common/menu';
 import { Command, CommandRegistry } from '../common/command';
 import { DisposableCollection } from '../common/disposable';
-
-export interface AuthenticationSessionsChangeEvent {
-    added: ReadonlyArray<string>;
-    removed: ReadonlyArray<string>;
-    changed: ReadonlyArray<string>;
-}
+import {AuthenticationProviderAuthenticationSessionsChangeEvent} from '@theia/plugin';
+import * as theia from '@theia/plugin';
```

Comment:
@PhilippeVienne `@theia/core` should not depend on `@theia/plugin` (core is the parentmost extension).

Reply:
So I may duplicate Interfaces here from the VSCode API ?

Reply:
In fact there was previously a duplication, as we need to transport the same objects, I just corrected the code.